core/message.proto

```diff
@@ -5,7 +5,7 @@ message serializedMessage {
         required uint64 StreamID = 1;
         required uint64 PrevStreamID = 2;
         required int64 Timestamp = 3;
-        required uint64 Sequence = 4;
+        //required uint64 Sequence = 4;
```

Comment:
I'll suggest we leave that one here because this is the breaking change mentioned in my initial comment. We might still require it in the future.